src/utils/importPaths.ts

```diff
@@ -107,3 +98,12 @@ export const WM_WRITE_FELT: [string[], string] = [[...WARPLIB_MEMORY], 'wm_write
 
 export const ARRAY_TRAIT: [string[], string] = [['array'], 'ArrayTrait'];
 export const U256_FROM_FELTS: [string[], string] = [['warplib', 'integer'], 'u256_from_felts'];
+export const UINT256: [string[], string] = [
```

Comment:
I would use a different name for this import, maybe something like `GET_U128` as the name of the function is pretty long.
There are many places in the code that uses `UINT256` import, I don't think this is what it is expected.

Reply:
Actually that is expected, since it has to be imported everytime u256 struct is being constructed. I agree that's ugly and hackish, but as I suggested above I'd see a separate PR dealing with that. The function shouldn't be used, thus there is no point in debating over imports right now, we'll just remove them once we stop using it

Reply:
Name of constant changed